src/leaflet.filelayer.js

```diff
@@ -209,6 +210,48 @@
             }
             return layer;
         },
+		
+		/* Convert .poly into geojson and load it
+		 * The Osmosis polygon filter file format is supported by Osmosis, mapsplit, 
+		 * osmconvert, osmchange and pbftoosm as a way of defining extraction polygons.
+		 * https://wiki.openstreetmap.org/wiki/Osmosis/Polygon_Filter_File_Format
+		 */
+        _loadPoly: function _loadPoly(content) {
+			var lines = content.split("\n");
+			var json = {
+				"type": "Feature",
+				"name": $.trim(lines[0]),
+				"geometry": {
+					"type": "MultiPolygon",
+					"coordinates": [],
+					"crs":{"type":"name","properties":{"name":"EPSG:4326"}}
+				}
+			};
+			lines.splice(0, 1);
+			lines.splice(lines.length-1, 1);
+			
+			var begin = true;
+			var current = [];
+			lines.forEach(function (elt) {
+				var line =$.trim(elt);
+				line = line.replace(/\s+/g, ' ');
+				if (begin === true) {
+					begin = false;
+					return;
+				} else {
+					var coords = line.split(' ');
+					if (coords.length >= 2)	{
+						current.push([Number(coords[0]), Number(coords[1])]);
+					}
+				}
+				if (line === 'END') {
+					json.geometry.coordinates.push([current]);
+					current = [];
+					begin = true;
+				}
+			});
```

Comment:
I would suggest that you create a dedicated function `poly2geojson()` with some unit tests :)

Reply:
Well, it seems to be for me the most complex part... Do I really need to install nodejs and all thoses things called chai mocha simon happen ? I tryed npm, but I get various sorts of errors I don't understand : like "Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})"... I'm surprised that Debian Linux 9.6 on x64 is not supported by mocha... and I don't have time to search why all those tangled things don't work. Maybe you have a simple procedure?

Reply:
Also I'm using a virtualized file system and it doesn't seem compatible with npm. It says: 
```
npm ERR! rofs EROFS: read-only file system, symlink '../acorn/bin/acorn' -> '/media/sf_www/kikiminou/Leaflet.FileLayer/node_modules/.bin/acorn'
npm ERR! rofs Often virtualized file systems, or other file systems
npm ERR! rofs that don't support symlinks, give this error.
```

Reply:
May be I can simply create the entry poly2geojson in test.filelayer.js and let someone else test ?

Reply:
>  I don't have time to search why all those tangled things don't work

I know exactly what you mean. And me neither :) 

> Maybe you have a simple procedure?

Using [nvm](https://github.com/creationix/nvm) is usually helpful to get a recent version of npm working. 

> Maybe I can simply create the entry poly2geojson in test.filelayer.js and let someone else test it ?

You could write the test and let the Continous Integration run it for you when you push to your branch :) Look, that's the output of your last commit https://travis-ci.org/makinacorpus/Leaflet.FileLayer/builds/496427902 

Merging without tests is possible too, if folks from @makinacorpus approve it :)